### PR TITLE
Wormhole shim implementation

### PR DESCRIPTION
**Pull request by mjbrisebois**
This is the shim solution from [DL-H0005](https://github.com/Holo-Host/rfcs/blob/master/teams/holo-hosting/decision_logs/DL-H0005-wormhole-updates-rsm.md)

Lair must be running for tests
```
nix-shell
make lair
```

The pre-existing e2e tests are not working yet so you must run the wormhole tests in isolation
```
LOG_LEVEL=silly npx mocha tests/e2e/test_wormhole.js
```

### CODE REVIEW

File: build/wormhole.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	    "parser": parser,
+	});
+
+	lair_stream.pipe( conductor_stream );
+	conductor_stream.pipe( parser );
+
+	for await ( let header of parser ) {
+	    if ( header === null )
+		continue;
+		// throw new Error(`Got a null header`);
```
Comment (robbiecarlton) [inline build/wormhole.js]: Remove comment

File: tests/e2e/test_wormhole.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	    shim_client.destroy();
+	}
+
+	expect( resp.get(0)		).to.be.a('number');
+	expect( resp.get(1)		).to.be.a('uint8array');
+	expect( resp.get(2)		).to.be.a('uint8array');
+	expect( recv_unlock		).to.be.true;
+    });
+
+    it("should process request and respond", async () => {
```
Comment (robbiecarlton) [inline tests/e2e/test_wormhole.js]: This test and the previous one have the same name. Can you give them different names?

**Review by robbiecarlton — CHANGES_REQUESTED** [review]
Couple of small changes, otherwise looks good.

File: build/wormhole.js (hunk 1 above)
Comment (mjbrisebois) [reply build/wormhole.js]:
```suggestion
```

**Review by robbiecarlton — APPROVED** [review]

**Pull request merged**